src/fastoad/_utils/sellar/function_f.py

```diff
@@ -35,8 +35,8 @@ def compute(self, inputs, outputs, discrete_inputs=None, discrete_outputs=None):
         """Functions computation"""
 
         z2 = inputs["z"][1]
-        x1 = inputs["x"]
-        y1 = inputs["y1"]
-        y2 = inputs["y2"]
+        x1 = inputs["x"].item()
+        y1 = inputs["y1"].item()
+        y2 = inputs["y2"].item()
```

Comment:
Two questions on that:
- Why do we need to use .items() now ?
- Shouldn't that be done in all the FAST-OAD components then ?

Reply:
Actually, I decided to do these changes in the mission module (in a pending branch), to ensure working with floats instead of 1-element arrays. The latter needed special attention before writing csv file, plus more CPU time for arithmetical operations.
The "obvious" way to have floats would be `[0]`, but numpy provides `.item()` that has some advantages, like ensuring the target is actually a 1-element array, and working alike on `np.array([0.])` and `np.array(0.)` (though the latter is not that useful in our context)